portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,123 @@
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <div id="top">
+        <nav class="navbar navbar-expand-lg fixed-top navbar-light">
+            <a class="navbar-brand nav-item" href="#top"><h3>RG</h3></a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                <ul class="navbar-nav">
+                    <li class="nav-item activate">
+                        <a class="nav-link" href="#">About me</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Experiences</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Hobbies</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Contact</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <div class="bg-image"></div>
+        <div class="bg-text">
+            <h2>Rishika Gupta</h2>
+            <h1 style="font-size:50px">A Problem solver</h1>
+            <p>I am currently in my 4th year of computer engineering at Indira Gandhi Delhi Technical University for Women </p>
+        </div>
+        <div class="container2">
+            <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
+                <div class="carousel-inner">
+                    <div class="carousel-item active">
+                        <div class="container justify-content-around">
+                            <h2 class="text-type">LinkedIn Wintathon experience</h2>
+                            <div class="row d-flex justify-content-around">
+                                <div class="col-12 col-lg-4">
+                                    <div class="card">
+                                        <img src="images/Wintathon1.JPG" class="card-img-top" alt="...">
```

Comment:
Not coding related but on performance: some of these images are really large in size. most of the time it is possible to reduce the size without significant impact on quality